.dockerignore

```diff
@@ -1,3 +1,3 @@
 src/**/*.test.ts
-src/**/*.test.tsx
+src/**/*.test.ts
```

Comment:
This looks erroneous. Tsx tests should be ignored and ts tests are already ignored above.